### PR TITLE
add erdmannfreunde/theme-toolbox

### PR DESCRIPTION
**Pull request by denniserdmann**

### CODE REVIEW

File: meta/erdmannfreunde/theme-toolbox/en.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+en:
+    title: Theme Toolbox
+    description: Create sets of readable css classes, which can be used by editors in articles, content elements, frontend modules, news, events and faq. On top of that you can disable the asset cache of contao, in order to watch changes in scss files immediately after refreshing your browser.
```
Comment (xchs) [inline meta/erdmannfreunde/theme-toolbox/en.yml:3]:
```suggestion
    description: Create sets of readable css classes, which can be used by editors in articles, content elements, frontend modules, news, events and FAQ. On top of that you can disable the asset cache of Contao, in order to watch changes in SCSS files immediately after refreshing your browser.
```

**Review by xchs — COMMENTED** [review]
There are some issues that should be addressed before this can be merged:

image

**Comment by denniserdmann** [comment]
@xchs Done!

**Comment by xchs** [comment]
> @xchs Done!

Where did you make your additions? Did you commit the changes?

**Comment by denniserdmann** [comment]
see #506

**Comment by xchs** [comment]
@denniserdmann The changes should be made for this PR. May I ask you that you commit your changes into this PR? Thanks.

**Review by Toflar — APPROVED** [review]

**Pull request merged**